soc/arm/actions/atb110x/boot_header.ld

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2019 Manivannan Sadhasivam <mani@kernel.org>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+. = 0xc0;
+LONG(0x00425441);
```

Comment:
Does this magic number having meaning?  If so a comment about it.

Reply:
Added